<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueueManagerDialog</class>
 <widget class="QDialog" name="QueueManagerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Queue Manager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="queueTable">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonBox">
     <item>
      <widget class="QPushButton" name="addQueueButton">
       <property name="toolTip">
        <string>Add a new queue.</string>
       </property>
       <property name="text">
        <string>&amp;Add...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="configureQueueButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Configure and add programs to the selected queue.</string>
       </property>
       <property name="text">
        <string>&amp;Configure...</string>
       </property>
       <property name="Enabled" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeQueueButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Remove the selected queue.</string>
       </property>
       <property name="text">
        <string>&amp;Remove</string>
       </property>
       <property name="Enabled" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="importQueueButton">
       <property name="text">
        <string>Import...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportQueueButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Export...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>queueTable</tabstop>
  <tabstop>addQueueButton</tabstop>
  <tabstop>configureQueueButton</tabstop>
  <tabstop>removeQueueButton</tabstop>
  <tabstop>importQueueButton</tabstop>
  <tabstop>exportQueueButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
